src/pages/billing/CreateAzureBillingProjectModal.test.js

```diff
@@ -24,11 +25,17 @@ jest.mock('src/components/Modal', () => {
 })
 jest.mock('src/libs/ajax')
 
+expect.extend(toHaveNoViolations)
```

Comment:
Could this go in src/setupTests.js to avoid needing to repeat the setup in each test?

Reply:
Thanks for the suggestion, I wasn't aware of that file!